docs/csharp/methods.md

```diff
@@ -34,7 +34,7 @@ This topic contains the following sections:
 - [Expression-bodied members](#expr)
 - [Iterators](#iterators)
 
-<a name="signatures" /a>
+<a name="signatures" />
```

Comment:
... and there are those handful of others in the doc. Tag-closing them is safe. I recommend it. Who knows what kind of crazy browsers out there might try to parse this and choke. 😨 Alternatively, we could track down links to these anchors and convert them over to headings.